tests/Validator/PossibleTypeExtensionsTest.php

```diff
@@ -0,0 +1,437 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
```suggestion
<?php declare(strict_types=1);
```

Reply:
Not sure how this happened, my PHPStorm's settings had to get messed up, I normally use a single line for this.  Why php-cs-fixer doesn't report/fix it?

Reply:
There is no setting to enforce this, it is only possible for now to have php-cs-fixer *not* mess it up.